hi-ml-histopathology/src/histopathology/models/deepmil.py

```diff
@@ -317,22 +312,11 @@ def __init__(self, tiles_count: int, **kwargs: Any) -> None:
         self.tiles_count = tiles_count
         super().__init__(**kwargs)
 
-    def _compute_bag_labels_logits_and_attn_maps(self, batch: Dict) -> Tuple[Tensor, Tensor, List]:
-        bag_labels_list = []
-        bag_logits_list = []
-        bag_attn_list = []
-        for bag_idx in range(len(batch[self.label_column])):
-            images = batch[TilesDataset.IMAGE_COLUMN][bag_idx]
-            label = batch[self.label_column][bag_idx]
-            bag_labels_list.append(label)  # no need to do majority voting to get the bag_label
-            logit, attn = self(images)
-            bag_logits_list.append(logit.view(-1))
-            bag_attn_list.append(attn)
-        bag_logits = torch.stack(bag_logits_list)
-        bag_labels = torch.stack(bag_labels_list).view(-1)
-        return bag_logits, bag_labels, bag_attn_list
+    @staticmethod
+    def get_bag_label(labels: Tensor) -> Tensor:
+        return labels
```

Comment:
Great! Minor suggestion to add a comment clarifying why here we return just `labels` for the slides pipeline.